src/components/buttons/button-image.jsx

```diff
@@ -87,7 +87,7 @@ class ButtonImage extends React.Component {
 	 * @method _onInputChange
 	 * @protected
 	 */
-	_onInputChange() {
+	_onInputChange = () => {
```

Comment:
Perhaps unsurprisingly, the button that hadn't been used for months wasn't actually working, because `this` wasn't bound.